src/main/scala/mesosphere/marathon/state/EntityStoreCache.scala

```diff
@@ -83,6 +83,7 @@ class EntityStoreCache[T <: MarathonState[_, T]](store: EntityStore[T])
 
     def handleEntries(names: Seq[String]): Future[Unit] = {
       val (unversionedNames, versionedNames) = names.partition(noVersionKey)
+      log.debug(s"$store Preload and cache entries: $unversionedNames and versioned entries $versionedNames")
```

Comment:
These can result in a lot of "names". So please wrap the debug statement in an `if (log.isDebugEnabled)`.

(We should really switch to suitable Scala logging macros)